lib/analysis_server.dart

```diff
@@ -249,7 +250,7 @@ class AnalysisServer implements Disposable {
     if (isActive) _server.analysis.reanalyze();
   }
 
-  Stream<SearchResult> filterSearchResults(String id) {
+  Stream<SearchResult> _filterSearchResults(String id) {
```

Comment:
_filterSearchResults --> _searchResultsStream ?
That way it would align nicely with the rename suggested above.